internal/route/routes.go

```diff
@@ -28,9 +30,10 @@ func HandleIndex(w http.ResponseWriter, r *http.Request) {
 
 		if err := r.ParseForm(); err != nil {
 			fmt.Fprintf(w, "ParseForm() err: %v", err)
+			return
 		}
 		var todos []string
-		todo = r.FormValue("todolist")
+		tplvars.Todo = r.FormValue("todolist")
```

Comment:
This is how we can set the vars. Makes it obvious that we're going to use them in the template too!